### ISSUE TITLE
error when running eval_linear.py

### ISSUE THREAD

**Issue by meryusha** [kahnchana/svt#3]
Dear authors! Thanks a lot for publishing the code

I encounter the following error when running the code:
![image](https://user-images.githubusercontent.com/20293224/171167644-ce4773dd-3071-48d7-860a-34d17e78c943.png)
It seems that the input is list of global and local views and only one input is expected to be passed? 
Could you please help me with it?

**Comment by meryusha** [comment]
doing` inp = inp[0].cuda(non_blocking=True)` works but i am not sure if that is the proper way

**Comment by kahnchana** [comment]
Thanks a lot for your interest in our work. 

Does this happen only for Kinetics dataset? Or for UCF / HMDB as well? 

It seems this error happened with a later edit I did in the dataset classes, will try to fix it soon. For linear eval training, I only used a single view at each train step (so what you did should work as a quick fix).  

For slow-fast inference, I was updating this to process two parallel views and average the embedding during training and inference. In that case `inp = [x.cuda(non_blocking=True) for x in inp]` should work.
